src/components/Form/Form.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import './Form.scss';
+import usersFromServer from '../../api/users';
+import { Todo } from '../../types/Todo';
+import { getUserByTodo } from '../services/user';
+
+type Props = {
+  onSubmit: (post: Todo) => void;
+};
+
+export const Form: React.FC<Props> = ({ onSubmit }) => {
+  const [title, setTitle] = useState<string>('');
+  const [hasTitleError, setHasTitleError] = useState<boolean>(false);
+
+  const [userId, setUserId] = useState<number>(0);
+  const [hasSelectError, setHasSelectError] = useState<boolean>(false);
```

Comment:
In most case s typescript can guess the type, so it is ok to use following:

```suggestion
  const [title, setTitle] = useState('');
  const [hasTitleError, setHasTitleError] = useState(false);

  const [userId, setUserId] = useState(0);
  const [hasSelectError, setHasSelectError] = useState(false);
```